raiden-dapp/CHANGELOG.md

```diff
@@ -37,6 +38,7 @@
 [#1374]: https://github.com/raiden-network/light-client/issues/1374
 [#168]: https://github.com/raiden-network/light-client/issues/168
 [#1805]: https://github.com/raiden-network/light-client/issues/1805
+[#1876]: https://github.com/raiden-network/light-client/pull/1876
```

Comment:
I'm just curious: I thought you teach me that issues are better to link since they are more descriptive. Why did you mixed that here? :thinking:

Reply:
As I said there, we must be sensible about the context. If there's an issue which lists or describe, even in comments, the reason for a PR, the issue should be used as the changelog entry. If the issue doesn't describe at all what is more relevant for the changelog entry (in this case, the more important fix was on the dApp side, and the SDK side was more about making it more reliable), and it's not worth it to create a new issue just for the related change here, then the PR can be listed.